app/models/normalize_timdex.rb

```diff
@@ -33,10 +33,36 @@ def extract_results(results, norm)
 
   def extract_title(record)
     title = record['title']
-    title << (" (#{record['identifier'].to_s.gsub('MIT:archivespace:', '')})") if record['identifier'].present?
+    title << extract_collection_id(record['identifiers']) if record['identifiers'].present?
     title
   end
 
+  def extract_dates(dates)
+    # It is unlikely for a record to have more than one creation date, but just in case...
+    relevant_dates = dates.select { |date| date['kind'] == 'creation' }
+
+    # If the record *does* have more than one creation date, it's probably not worth determining which to display.
+    return if relevant_dates.count > 1
```

Comment:
I'm waffly on whether displaying no dates or the first date is better. (This is not suggesting a change, just saying that both paths seem equally valid here and thus this one is 👍🏻 )

Reply:
Yeah, I feel the same way. Of course, it's highly unlikely that this will be an issue, but now that I've said that...